add-link

```diff
@@ -38,3 +38,4 @@ https://www.mdscbenin.org/wp-includes/theme-compat/dar/newsant1/login.php
 https://nort2on.com/setup/
 http://eternalstarculture.com/intern/home.php
 https://tinyurl.com/4cbf4wcp
+https://www.kunestra.hr/.uk/
```

Comment:
Confirmed, but you should be aware it is also working on all four versions of 

```js
http(s)?://(www.)?kunestra.hr/.uk/
```

![image](https://user-images.githubusercontent.com/44526987/126137228-6db83872-6a6a-4b0b-86f5-ffbcb33a847d.png)